com.unity.ml-agents.extensions/Editor/GridSensorComponentEditor.cs

```diff
@@ -0,0 +1,140 @@
+using UnityEditor;
+using UnityEngine;
+using Unity.MLAgents.Editor;
+using Unity.MLAgents.Extensions.Sensors;
+
+namespace Unity.MLAgents.Extensions.Editor
+{
+    [CustomEditor(typeof(GridSensorComponent))]
+    [CanEditMultipleObjects]
+    internal class GridSensorComponentEditor : UnityEditor.Editor
+    {
+        const string k_SensorName = "m_SensorName";
```

Comment:
I know we don't do this elsewhere right now, but I think you can do e.g. `nameof(GridSensor.m_SensorName)` (although that might not work if it's private, so you'd have to change it to internal).

Unfortunately these string constants don't really do much, because they're not used in multiple places, and things will still break if the member gets renamed.

Reply:
changed it to use nameof()